src/Components/Components/src/Forms/InputComponents/InputNumber.cs

```diff
@@ -100,98 +78,23 @@ protected override string FormatValueAsString(T value)
                     return null;
 
                 case int @int:
-                    return @int.ToString(CultureInfo.InvariantCulture);
+                    return BindConverter.FormatValue(@int, CultureInfo.InvariantCulture);
```

Comment:
I'm assuming these changes are just so we can reason that all formatting is done inside `BindConverter.Format*` methods, since the resulting behavior would be identical even if this wasn't changed. Is that right?

Reply:
yes, exactly